contao/config/event_listeners.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * The MetaModels extension allows the creation of multiple collections of custom items,
+ * each with its own unique set of selectable attributes, with attribute extendability.
+ * The Front-End modules allow you to build powerful listing and filtering of the
+ * data in each collection.
+ *
+ * PHP version 5
+ *
+ * @package    MetaModels
+ * @subpackage AttributeText
+ * @author     Christopher Boelter <christopher@boelter.eu>
+ * @copyright  The MetaModels team.
+ * @license    LGPL.
+ * @filesource
+ */
+
+use MetaModels\Attribute\Events\CreateAttributeFactoryEvent;
+use MetaModels\Attribute\Text\AttributeTypeFactory;
+use MetaModels\MetaModelsEvents;
+use MetaModels\Events\MetaModelsBootEvent;
+use MetaModels\DcGeneral\Events\Table\InputScreens\BackendSubscriber;
```

Comment:
I am fine with that but all the other attributes currently keep everything below their namespace...
I guess we should write some generic docs and we should clean up the namespace mess.
Currently it is all pretty mixed up (at least everything backend related).